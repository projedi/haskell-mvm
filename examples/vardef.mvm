int i;
double j;
string str;

i = 0;
j = 3.1;
str = "abc";
print(i, " ", j, " ", str, "\n");
i = i + 1;
j = j * 1.4;
str = "def";
print(i, " ", j, " ", str, "\n");

i += 1;
j -= 1;
print(i, " ", j, "\n");
