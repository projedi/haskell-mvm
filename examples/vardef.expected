0 3.1 abc
1 4.34 def
2 3.34
